Set view as empty with double tap

Mauker1 opened the pull request:
Fix #91 
Updated the `mIsEmpty` flag once the user double taps the signature pad. 
Previous to this modification, if the user signed the view, and double tapped, the method `isEmpty()` would always return `false`.

Mauker1 committed:
Set view as empty with double tap

Updated the "mIsEmpty" flag once the user double taps the signature pad.

gcacace commented on signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java:
```diff
@@ -208,7 +208,10 @@ public boolean onTouchEvent(MotionEvent event) {
                 getParent().requestDisallowInterceptTouchEvent(true);
                 mPoints.clear();
-                if (isDoubleClick()) break;
+                if (isDoubleClick()) {
```
Shouldn't this be applicable only if the configuration for `clearOnDoubleClick` is `true`?

gcacace commented on signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java:
```diff
@@ -208,7 +208,10 @@ public boolean onTouchEvent(MotionEvent event) {
                 getParent().requestDisallowInterceptTouchEvent(true);
                 mPoints.clear();
-                if (isDoubleClick()) break;
+                if (isDoubleClick()) {
```
Can you extract from the function `isDoubleClick()` the part where the signature is cleared, so we can have the clearing part and the `setIsEmpty(true)` in the same place? I'd like to have `isDoubleClick()` as immutable method.

Mauker1 replied:
Sure. Can you look at the new commit to check if that's what you wanted?

Mauker1 committed:
Update SignaturePad.java

Removed `this.clearView();` from the `isDoubleClick()` method.
It's now placed inside `onTouchEvent()` if the `isDoubleClick()` condition is `true`.

Mauker1 commented:
I've tested it, and it's not working as it should yet because [this line](https://github.com/gcacace/android-signaturepad/pull/91/files/8ec384077d1ecd1b819a2ff05546a6445de3c074#diff-3378ae334d26c22774115b5b8783b0d2L226) keeps resetting the variable.

Aks-4125 reviewed (changes requested):
Not working.
